source/CollisionSet.cpp

```diff
@@ -72,6 +72,7 @@ namespace {
 // Initialize a collision set. The cell size and cell count should both be
 // powers of two; otherwise, they are rounded down to a power of two.
 CollisionSet::CollisionSet(unsigned cellSize, unsigned cellCount)
+	: seenEpoch(0)
```

Comment:
Let's initialize this in the header file.

Reply:
Done